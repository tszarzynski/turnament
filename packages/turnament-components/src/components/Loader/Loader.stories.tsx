import React from 'react';
import Loader from './Loader';

export default {
  component: Loader,
  title: 'Loader',
};

export const Default = () => {
  return <Loader />;
};
