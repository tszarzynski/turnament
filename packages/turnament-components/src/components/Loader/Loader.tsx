import React from 'react';
import { CircularProgress, Box } from '@material-ui/core';

const Loader: React.FC = () => {
  return (
    <Box
      display="flex"
      alignItems="center"
      justifyContent="center"
      width={1}
      minHeight="100vh"
    >
      <CircularProgress />
    </Box>
  );
};

export default Loader;
