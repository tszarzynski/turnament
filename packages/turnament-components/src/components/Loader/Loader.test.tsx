import React from 'react';
import { render } from '@testing-library/react'
import Loader from './Loader';

describe('Loader', () => {
   it('renders without error', () => {
      const { asFragment } = render(
         <Loader/>
       )
      expect(asFragment()).toMatchSnapshot()
   });
});