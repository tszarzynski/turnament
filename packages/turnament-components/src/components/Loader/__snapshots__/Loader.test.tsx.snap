// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader renders without error 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root MuiBox-root-2"
  >
    <div
      class="MuiCircularProgress-root MuiCircularProgress-colorPrimary MuiCircularProgress-indeterminate"
      role="progressbar"
      style="width: 40px; height: 40px;"
    >
      <svg
        class="MuiCircularProgress-svg"
        viewBox="22 22 44 44"
      >
        <circle
          class="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate"
          cx="44"
          cy="44"
          fill="none"
          r="20.2"
          stroke-width="3.6"
        />
      </svg>
    </div>
  </div>
</DocumentFragment>
`;
